helm/cluster/templates/_helpers.tpl

```diff
@@ -3,14 +3,14 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "name" -}}
+{{- define "cluster.chart.name" -}}
```

Comment:
Added `cluster` prefix to template names, as Helm recommends that to avoid template name conflicts.